File: app/src/main/java/com/example/movies/ui/actorsdetailtabs/view/MoviesFragment.java
package com.example.movies.ui.actorsdetailtabs.view;

import android.os.Bundle;

import com.example.movies.core.base.BaseActorsShowFragment;

public class MoviesFragment extends BaseActorsShowFragment {

    @Override
    public void fetchActorsShows() {
        service.fetchActorsMovie(id, this);
    }

    public static MoviesFragment newInstance(int id) {
        MoviesFragment moviesFragment = new MoviesFragment();
        Bundle bundle = new Bundle();
        bundle.putInt("id", id);
        moviesFragment.setArguments(bundle);
        return moviesFragment;
    }


}

File: app/src/main/java/com/example/movies/ui/actorsdetailtabs/view/MoviesFragment.kt
package com.example.movies.ui.actorsdetailtabs.view

import android.os.Bundle

import com.example.movies.core.base.BaseActorsShowFragment

class MoviesFragment : BaseActorsShowFragment() {

    override fun fetchActorsShows() {
        id?.let { id -> service.fetchActorsMovie(id, this) }

    }

    companion object {

        fun newInstance(id: Int): MoviesFragment {
            val moviesFragment = MoviesFragment()
            val bundle = Bundle()
            bundle.putInt("id", id)
            moviesFragment.arguments = bundle
            return moviesFragment
        }
    }


}
